modules/packages/LinearAlgebra.chpl

```diff
@@ -1900,6 +1900,20 @@ module Sparse {
     return S;
   }
 
+  proc _array.plus(A: [?Adom] ?eltType) where isSparseArr(this) && !isCSArr(this)
```

Comment:
Let's `pragma "no doc"` these implementations and modify the doc string on the original implementations to mention that these operations support COO and CSR arrays.